var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -36,18 +36,19 @@ class PyNumpy(PythonPackage):
     homepage = "http://www.numpy.org/"
     url      = "https://pypi.io/packages/source/n/numpy/numpy-1.9.1.tar.gz"
 
+    version('1.12.0', '33e5a84579f31829bbbba084fe0a4300',
+            url="https://pypi.io/packages/source/n/numpy/numpy-1.12.0.zip")
```

Comment:
The latest numpy release only comes with `.zip` files. I'm not sure if this will always be the case from now on, but do we want to switch all of the numpy URLs (and checksums) to `.zip`?